spec/requests/api/system/force_sync_spec.rb

```diff
@@ -7,6 +7,14 @@
   end
 
   before do
+    #The settings need to be reset as it has cached the qa settings (see
+    # db_spec.rb:25)
+    Rails.env = "test"
+    Settings.reload_from_files(
+      Rails.root.join("config", "settings.yml").to_s,
+      Rails.root.join("config", "settings", "#{Rails.env}.yml").to_s,
+      Rails.root.join("config", "environments", "#{Rails.env}.yml").to_s,
+    )
```

Comment:
Is this needed because the last test to run is `db_spec.rb`? If so, then this is the wrong way to fix it I think, because:

1. tests could run in a different order, which would mean that a different test could fail (because the Settings are still set to `qa`)
2. other tests could also invoke the `env_to_azure_map` code path, which will break some other tests

I think we the best way to handle this is to stub out the bit that reloads settings in `env_to_azure_map`, it may not be clear how to do that so grab me next time I'm there, or maybe @danielburnley or @TobyRet, to help explain how that might be done.

Reply:
I think all the comments have now been addressed. Thanks for the help @misaka !